indexing-hadoop/pom.xml

```diff
@@ -67,6 +67,10 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-aws</artifactId>
```

Comment:
Agreed. But if Hadoop 2.6.0 is used during runtime, one will have to also include `hadoop-aws`. 

I will post this in Google groups.